### PR TITLE
Explain the device format in man pages

### PR DESCRIPTION
**Pull request by QiWang19**
Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

File: docs/podman-create.1.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Override the key sequence for detaching a container. Format is a single characte
 
 **--device**=[]
 
-Add a host device to the container (e.g. --device=/dev/sdc:/dev/xvdc:rwm)
+Add a host device to the container. The format is `<device-on-host>:<device-on-container>:<permissions>` (e.g. --device=/dev/sdc:/dev/xvdc:rwm)
```
Comment (mheon) [inline docs/podman-create.1.md]: I think that host device and permissions are both optional - maybe enclose them in square brackets to show that?

**Comment by rhatdan** [comment]
Here is what docker currently has.

```
          Add a host device onhost to the container under the incontainer name.  Optional mode
       parameter can be used to specify device permissions, it is a combination of r (for read),
       w (for write), and m (for mknod(2)).

       For example, --device=/dev/sdc:/dev/xvdc:rwm will give a container all permissions for the
       host device /dev/sdc, seen as /dev/xvdc inside the container.
```

File: docs/podman-create.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/podman-create.1.md]: Something like:
```
<devcie-on-host>[:<device-on-container>][:<permissions>]
```

Comment (TomSweeneyRedHat) [reply docs/podman-create.1.md]: I missed the optional part, nice catch.  I'm fine with @rhatdan's suggestion with the brackets above then.  The Docker variant would work too, but I kinda prefer the shorter version.

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
bot, retest this please

**Comment by baude** [comment]
please rebase

**Comment by mheon** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@QiWang19 Rebase this please and repush.  Something weird is going on.

**Comment by rhatdan** [comment]
bot, retest this please

**Pull request merged**